scripts/00_setup.R

```diff
@@ -1,6 +1,8 @@
 install.packages("groundhog", repos="https://ftp.fau.de/cran/")
 library("groundhog")
+# TODO: setwd() for this R session to scripts/? Or ask users to setwd() accordingly, otherwise groundhog & paths don't work as expected.
```

Comment:
As an r-user who may run scripts like this, I prefer it if people do not setwd and instead go from the assumption that the folder where most scripts are or the folder called "code" is the wd. Then I can set that however I'd like, with the here-package, r-project or setwd. So I'd go for the "tell user in README"-path if any.

Reply:
Genau, I do the same with Makefiles.